addnum.c

```diff
@@ -0,0 +1,11 @@
+#include<stdio.h>
```

Comment:
Your assignment3.txt is showing up as a binary file, so I can't view it, which means I can't grade it.